doc/status_change.txt

```diff
@@ -2528,6 +2528,55 @@ SC_CHASEWALK2	(SI_CHASEWALK2)
 	desc: 2nd effect of Chasewalk
 	val1: +STR
 
+SC_GVG_GIANT	(SI_GVG_GIANT)
+	desc: Instantly consumes some amount of HP, increases Physical/Magic damage on player enemies by n%.
```

Comment:
Description only says HP, but val2 also has SP(same for all of those).
Maybe you could also refer to the values like:
Instantly consumes <val1> HP, increases Physical/Magical damage on player enemies by <val3>/<val4>%.